xarray/tests/test_backends.py

```diff
@@ -1384,6 +1384,7 @@ def test_replace_slices_with_arrays():
 
 
 @requires_scipy
+@requires_dask
```

Comment:
This isn't quite right -- we still want to check the reading/writing a netCDF file with SciPy work without dask. I'm going to look alternate fixes for the `can't pickle _thread.lock objects` issue.

Reply:
OK! I'll leave this open for the moment